ground-core/src/main/java/edu/berkeley/ground/api/models/EdgeFactory.java

```diff
@@ -14,18 +14,17 @@
 
 package edu.berkeley.ground.api.models;
 
-import edu.berkeley.ground.db.DBClient.GroundDBConnection;
-import edu.berkeley.ground.exceptions.GroundException;
+import edu.berkeley.ground.exceptions.GroundDBException;
 
 import java.util.List;
 import java.util.Map;
 
 public abstract class EdgeFactory {
-  public abstract Edge create(String name, Map<String, Tag> tags) throws GroundException;
+  public abstract Edge create(String name, Map<String, Tag> tags) throws GroundDBException;
```

Comment:
Hmm fair enough, I was just trying to unify the use of the DB exceptions, as they were GroundException in one spot and GroundDBException in another spot (sometimes in the same file).